README.md

```diff
@@ -19,14 +19,14 @@ Windows), mobile platforms (Android, iOS), as well as Web-based platforms
 
 ## Free, open source and community-driven
 
-Godot is completely free and open source under the very permissive [MIT license](https://godotengine.org/license).
+Godot is completely free, and open-source under the very permissive [MIT license](https://godotengine.org/license).
```

Comment:
I think adding a comma before `and` doesn't feel grammatically correct for me.

Reply:
I'd also keep that sentence as it is currently. Having too many commas can reduce readability.